lib/widgets/fields/padding_field/padding_text_fields.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+class PaddingTextField extends StatelessWidget {
```

Comment:
The name can reflect something more useful maybe. Like what is the behavior of this TextField that makes it different from the other implementations? Maybe `BorderlessTextField`?